backend/business-partner-agent/src/main/java/org/hyperledger/bpa/repository/MyCredentialRepository.java

```diff
@@ -55,4 +55,9 @@
 
     Long countByStateEquals(CredentialExchangeState state);
 
+    @Query("SELECT * FROM my_credential WHERE type = 'SCHEMA_BASED' AND referent IS NOT NULL AND (revoked IS NULL OR revoked = false)")
```

Comment:
Not related to this, but because I just stumbled over this. We should keep in mind to rename "SCHEMA_BASED" to "INDY".